README.md

```diff
@@ -1 +1,276 @@
+      #### OpenTofu Initialization ⚙️\`${{ steps.init.outcome }}\`
+      #### OpenTofu Validation 🤖\`${{ steps.validate.outcome }}\`
+      <details><summary>Validation Output</summary>
+
+      \`\`\`\n
+      ${{ steps.validate.outputs.stdout }}
+      \`\`\`
+
+      </details>
+
+      #### OpenTofu Plan 📖\`${{ steps.plan.outcome }}\`
+
+      <details><summary>Show Plan</summary>
+
+      \`\`\`\n
+      ${process.env.PLAN}
+      \`\`\`
+
+      </details>
+
+      *Pusher: @${{ github.actor }}, Action: \`${{ github.event_name }}\`, Working Directory: \`${{ env.tf_actions_working_dir }}\`, Workflow: \`${{ github.workflow }}\`*`;
+
+      // 3. If we have a comment, update it, otherwise create a new one
+      if (botComment) {
+        github.rest.issues.updateComment({
+          owner: context.repo.owner,
+          repo: context.repo.repo,
+          comment_id: botComment.id,
+          body: output
+        })
+      } else {
+        github.rest.issues.createComment({
+          issue_number: context.issue.number,
+          owner: context.repo.owner,
+          repo: context.repo.repo,
+          body: output
+        })
+      }
+```
+
+## Inputs
+
+The action supports the following inputs:
+
+- `cli_config_credentials_hostname` - (optional) The hostname of a Terraform Cloud/Enterprise instance to
+  place within the credentials block of the OpenTofu CLI configuration file. Defaults to `app.terraform.io`.
+- `cli_config_credentials_token` - (optional) The API token for a Terraform Cloud/Enterprise instance to
+  place within the credentials block of the OpenTofu CLI configuration file.
+- `tofu_version` - (optional) The version of OpenTofu CLI to install. Instead of a full version string,
+  you can also specify a constraint string (see [Semver Ranges](https://www.npmjs.com/package/semver#ranges)
+  for available range specifications). Examples are: `<1.6.0-beta`, `~1.6.0-alpha`, `1.6.0-alpha2` (all three installing
+  the latest available `1.6.0-alpha2` version). Prerelease versions can be specified and a range will stay within the
+  given tag such as `beta` or `rc`. If no version is given, it will default to `latest`.
+- `tofu_wrapper` - (optional) Whether to install a wrapper to wrap subsequent calls of
+  the `tofu` binary and expose its STDOUT, STDERR, and exit code as outputs
+  named `stdout`, `stderr`, and `exitcode` respectively. Defaults to `true`.
+
+## Outputs
+
+This action does not configure any outputs directly. However, when you set the `tofu_wrapper` input
+to `true`, the following outputs are available for subsequent steps that call the `tofu` binary:
+
+- `stdout` - The STDOUT stream of the call to the `tofu` binary.
+- `stderr` - The STDERR stream of the call to the `tofu` binary.
+- `exitcode` - The exit code of the call to the `tofu` binary.
+
+## License
+
+[Mozilla Public License v2.0](LICENSE)
+
+## Code of Conduct
+
+[Code of Conduct](CODE_OF_CONDUCT.md)
+
+## Experimental Status
```

Comment:
Maybe we should move this higher up near the top? Just so it's clear for everyone.

Reply:
Hey! Sure, I’ll move it to the top. For context, I preserved the structure of the readme from setup-terraform.

Reply:
Implemented.